src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -759,12 +790,17 @@ private void VerifyRanOnBackgroundThread(Times expected)
 
         private void VerifyServiceGetIssues(Times expected)
         {
-            mockSqService.Verify(x => x.GetSuppressedIssuesAsync(It.IsAny<string>(), null, It.IsAny<CancellationToken>()), expected);
+            mockSqService.Verify(x => x.GetSuppressedIssuesAsync(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<CancellationToken>()), expected);
         }
 
         private void VerifyServiceGetIssues(Times expected, string sonarQubeProjectKey)
         {
-            mockSqService.Verify(x => x.GetSuppressedIssuesAsync(sonarQubeProjectKey, null, It.IsAny<CancellationToken>()), expected);
+            mockSqService.Verify(x => x.GetSuppressedIssuesAsync(sonarQubeProjectKey, It.IsAny<string>(), It.IsAny<CancellationToken>()), expected);
```

Comment:
Miscellaneous: same again - this method is only checking the project key, and doesn't care about the `branch` parameter.